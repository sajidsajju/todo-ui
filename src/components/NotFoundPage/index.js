import React from "react";
import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles({
  wrapper: {
    height: "100vh",
    display: "flex",
    alignItems: "center",
    justifyContent: "center",
    flexDirection: "column",
    minHeight: "320px",
  },
  title: {
    marginTop: "-8vh",
    fontWeight: "bold",
    fontSize: "3.375rem",
  },
  span: {
    fontSize: "3.125rem",
  },
});
const NotFoundPage = () => {
  const classes = useStyles();

  return (
    <div className={classes.wrapper}>
      <div className={classes.title}>
        4
        <span className={classes.span} role="img" aria-label="Crying Face">
          ðŸ˜¢
        </span>
        4
      </div>
      <span className={classes.span}>Page not found</span>
    </div>
  );
};

export default NotFoundPage;
