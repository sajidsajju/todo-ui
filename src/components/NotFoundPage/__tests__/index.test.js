import React from "react";
import { render, screen } from "@testing-library/react";

import NotFoundPage from "../";

describe("<NotFoundPage  />", () => {
  it("should match the snapshot", () => {
    const rendered = render(<NotFoundPage />);

    expect(rendered.container.firstChild).toMatchSnapshot();
  });

  it("should match the Page not found text", () => {
    render(<NotFoundPage />);

    expect(screen.getByText("Page not found")).toBeInTheDocument();
  });
});
