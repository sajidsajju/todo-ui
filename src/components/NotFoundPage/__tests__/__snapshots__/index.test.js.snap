// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage  /> should match the snapshot 1`] = `
<div
  class="makeStyles-wrapper-1"
>
  <div
    class="makeStyles-title-2"
  >
    4
    <span
      aria-label="Crying Face"
      class="makeStyles-span-3"
      role="img"
    >
      ðŸ˜¢
    </span>
    4
  </div>
  <span
    class="makeStyles-span-3"
  >
    Page not found
  </span>
</div>
`;
